src/clr-angular/popover/common/popover.spec.ts

```diff
@@ -63,141 +63,141 @@ describe('Popover', function() {
 
     popover.style.transform = 'none';
     popoverInstance.anchor(anchor, Point.BOTTOM_CENTER, Point.TOP_RIGHT);
     x = padding + anchorWidth / 2 - popoverWidth - popoverMargin;
-    expect(popover.style.transform).toEqual(`translateX(${x}px) translateY(${y}px)`);
+    expect(popover.style.transform).toEqual(`translateX(${Math.round(x)}px) translateY(${Math.round(y)}px)`);
 
     popover.style.transform = 'none';
     popoverInstance.anchor(anchor, Point.BOTTOM_LEFT, Point.TOP_LEFT);
     x = padding;
-    expect(popover.style.transform).toEqual(`translateX(${x}px) translateY(${y}px)`);
+    expect(popover.style.transform).toEqual(`translateX(${Math.round(x)}px) translateY(${Math.round(y)}px)`);
 
     popover.style.transform = 'none';
     popoverInstance.anchor(anchor, Point.BOTTOM_LEFT, Point.TOP_RIGHT);
     x = 0;
-    expect(popover.style.transform).toEqual(`translateX(${x}px) translateY(${y}px)`);
+    expect(popover.style.transform).toEqual(`translateX(${Math.round(x)}px) translateY(${Math.round(y)}px)`);
 
     // popovers on the left side of the anchor (some are covered already by above, so omitted)
     x = 0;
 
     popover.style.transform = 'none';
     popoverInstance.anchor(anchor, Point.LEFT_BOTTOM, Point.BOTTOM_RIGHT);
     y = padding + anchorHeight - popoverHeight - popoverMargin * 2;
-    expect(popover.style.transform).toEqual(`translateX(${x}px) translateY(${y}px)`);
+    expect(popover.style.transform).toEqual(`translateX(${Math.round(x)}px) translateY(${Math.round(y)}px)`);
 
     popover.style.transform = 'none';
     popoverInstance.anchor(anchor, Point.LEFT_CENTER, Point.RIGHT_TOP);
     y = padding + anchorHeight / 2 - popoverMargin;
-    expect(popover.style.transform).toEqual(`translateX(${x}px) translateY(${y}px)`);
+    expect(popover.style.transform).toEqual(`translateX(${Math.round(x)}px) translateY(${Math.round(y)}px)`);
 
     popover.style.transform = 'none';
     popoverInstance.anchor(anchor, Point.LEFT_CENTER, Point.RIGHT_CENTER);
     y = padding + anchorHeight / 2 - popoverHeight / 2 - popoverMargin;
-    expect(popover.style.transform).toEqual(`translateX(${x}px) translateY(${y}px)`);
+    expect(popover.style.transform).toEqual(`translateX(${Math.round(x)}px) translateY(${Math.round(y)}px)`);
 
     popover.style.transform = 'none';
     popoverInstance.anchor(anchor, Point.LEFT_CENTER, Point.RIGHT_BOTTOM);
     y = padding + anchorHeight / 2 - popoverHeight - popoverMargin;
-    expect(popover.style.transform).toEqual(`translateX(${x}px) translateY(${y}px)`);
+    expect(popover.style.transform).toEqual(`translateX(${Math.round(x)}px) translateY(${Math.round(y)}px)`);
 
     popover.style.transform = 'none';
     popoverInstance.anchor(anchor, Point.LEFT_TOP, Point.TOP_RIGHT);
     y = padding;
-    expect(popover.style.transform).toEqual(`translateX(${x}px) translateY(${y}px)`);
+    expect(popover.style.transform).toEqual(`translateX(${Math.round(x)}px) translateY(${Math.round(y)}px)`);
```

Comment:
And this is why we need to have better test helpers. 😂 

I know it was just a simple search & replace, but it still feels bad.